test/integration/test_mms.cpp

```diff
@@ -75,7 +75,7 @@ TEST(MMSTest, TwoStreamGrowthRate) {
   // that the run finishes as the
   // instability saturates.
   Mesh mesh(128, 0.05, 40);
-  Species electrons(mesh, true, 2.0, 1, 1, 12800);
+  Species electrons(mesh, true, 2.0, 1, 1, 14600);
```

Comment:
Any reason for this number in particular, other than 'it works'? I think it was 100 particles per mesh interval before.

Reply:
Might be worth adding a quick comment to that effect?